src/cmd/package.go

```diff
@@ -344,6 +344,11 @@ func bindCreateFlags(v *viper.Viper) {
 	createFlags.BoolVarP(&pkgConfig.CreateOpts.ViewSBOM, "sbom", "s", v.GetBool(common.VPkgCreateSbom), lang.CmdPackageCreateFlagSbom)
 	createFlags.StringVar(&pkgConfig.CreateOpts.SBOMOutputDir, "sbom-out", v.GetString(common.VPkgCreateSbomOutput), lang.CmdPackageCreateFlagSbomOut)
 	createFlags.BoolVar(&pkgConfig.CreateOpts.SkipSBOM, "skip-sbom", v.GetBool(common.VPkgCreateSkipSbom), lang.CmdPackageCreateFlagSkipSbom)
+	validateSchema := true
+	if v.Get(common.VPkgCreateValidateSchema) != nil {
+		validateSchema = v.GetBool(common.VPkgCreateValidateSchema)
+	}
+	createFlags.BoolVar(&pkgConfig.CreateOpts.ValidateSchema, "validate-schema", validateSchema, lang.CmdPackageCreateFlagValidateSchema)
```

Comment:
Went back and forth on whether I should add this flag, want to hear teams thoughts. In theory we have yaml extension covered so we shouldn't need an escape hatch. Not sure if we want this just in case tho.